integration-tests/src/test/java/oracle/kubernetes/operator/utils/DbUtils.java

```diff
@@ -230,16 +217,84 @@ public static void deleteNamespace(String namespace) throws Exception {
+   * @param namespace namespace to create
+   * @throws Exception if any error occurs
    */
   public static void createNamespace(String namespace) throws Exception {
     if (!namespace.equalsIgnoreCase("default")) {
-      String command = "kubectl create ns " + namespace;
-      logger.info("Running " + command);
-      TestUtils.execOrAbortProcess(command);
+      String cmd1 = "kubectl delete ns " + namespace + " --ignore-not-found";
+      logger.info("Running " + cmd1);
+      TestUtils.execOrAbortProcess(cmd1, true);
+      String cmd2 = "kubectl create ns " + namespace;
+      logger.info("Running " + cmd2);
+      TestUtils.execOrAbortProcess(cmd2, true);
+    }
+  }
+  
+  /**
+   * Start DB instance, create Oracle rcu pod and load database schema in the specified namespace.
+   * 
+   * @param scriptsDir directory of scripts
+   * @param dbPort NodePort of DB
+   * @param dbUrl URL of DB
+   * @param rcuSchemaPrefix rcu SchemaPrefixe
+   * @param namespace namespace where DB and RCU schema are going to start
+   * @throws Exception if any error occurs when setting up RCU database
+   */
+  public static void setupRcuDatabase(String scriptsDir, int dbPort, String dbUrl, 
+      String rcuSchemaPrefix, String namespace) throws Exception {  
+    
+    createNamespace(namespace);
+    createDockerRegistrySecret(namespace);
+    /*
+    //delete leftover pods caused by test being aborted
+    deleteRcuPod(scriptsDir);
+    deleteDbPod(scriptsDir);
+    
+    startOracleDB(scriptsDir, String.valueOf(dbPort), namespace);
+    createRcuSchema(scriptsDir,rcuSchemaPrefix, dbUrl, namespace);
+    */
+    String cmd1 = "mkdir -p " + scriptsDir
+        + "/scripts/create-rcu-schema/rcuoutput";
+    TestUtils.execOrAbortProcess(cmd1);
+    String cmd2 = "cp " 
+        + scriptsDir
+        + "/scripts/create-rcu-schema/common/rcu.yaml "
+        + scriptsDir
+        + "/scripts/create-rcu-schema/rcuoutput/";
+    TestUtils.execOrAbortProcess(cmd2);
+    createDBandRCUschema(scriptsDir, dbPort, dbUrl, rcuSchemaPrefix, namespace);
+    LoggerHelper.getLocal().log(Level.INFO,"RCU schema is going to create for:" 
+        + " namespace: " + namespace 
+        + " dbUrl:" + dbUrl 
+        + " dbPort: " + dbPort
+        + " rcuSchemaPrefix: " + rcuSchemaPrefix); 
+    
+  }
+  
+  private static void createDBandRCUschema(String scriptsDir, int dbPort, String dbUrl, 
+      String rcuSchemaPrefix, String namespace) throws Exception {
+    count++;
+    if (count < 2) {
+      deleteRcuPod(scriptsDir);
+      deleteDbPod(scriptsDir);
+      try {
+        startOracleDB(scriptsDir, String.valueOf(dbPort), namespace);
+        createRcuSchema(scriptsDir,rcuSchemaPrefix, dbUrl, namespace);
+        LoggerHelper.getLocal().log(Level.INFO,"RCU schema is created for:" 
+            + " namespace: " + namespace 
+            + " dbUrl:" + dbUrl 
+            + " dbPort: " + dbPort
+            + " rcuSchemaPrefix: " + rcuSchemaPrefix);
+        //TODO
```

Comment:
Describe what must still be done if you are leaving a TODO.

Reply:
Done